XMLHttpRequest/interfaces.html

```diff
@@ -166,6 +126,20 @@
     XMLHttpRequestUpload: ['(new XMLHttpRequest()).upload'],
     FormData: ['new FormData()', 'new FormData(form)']
   });
-});
-idlArray.test();
+  idlArray.test();
+}
+
+function waitForLoad() {
+  return new Promise(function(resolve) {
+    addEventListener("load", resolve);
+  });
+}
+
+promise_test(function() {
+  // Have to wait for onload
```

Comment:
Um, cargo-culting?  :)  (I don't know why the original html/dom/interfaces.html does either, but maybe it's so that all the objects are properly initialized.)  Fixed in new version.